src/dlite-bson.c

```diff
@@ -737,15 +737,15 @@ DLiteInstance *dlite_bson_load_instance(const unsigned char *doc)
     FAILCODE1(dliteKeyError, "expected dimension values to be a bson "
               "document, got %s", bson_typename(type));
   if ((ndims = bson_nelements(subdoc)) < 0) goto fail;
-  if (!(dims = calloc(ndims, sizeof(size_t))))
+  if (!(shape = calloc(ndims, sizeof(size_t))))
```

Comment:
Keep `dims`. This refer to the dimensions of the instance, not a property shape.

Reply:
done